components/document-table.js

```diff
@@ -40,6 +41,15 @@ const columns = [
     },
     sorter: (item) => item.attachment.download.toLowerCase(),
   },
+  {
+    id: "standardized_file_format",
+    title: "Standardized File Format",
+    display: ({ source }) => {
+      return source.standardized_file_format ? (
+        <Status status="standardized" />
+      ) : null;
+    },
+  },
```

Comment:
Does this column sort the way you want? When you sort ascending, all the ones with `standardized_file_format` true sort at the end.

Reply:
Should it simply be non-sortable?